frontend/components/TeamsDropdown/TeamsDropdown.tsx

```diff
@@ -66,26 +86,179 @@ const TeamsDropdown = ({
+          color: COLORS["core-vibrant-blue-over"],
+        },
+        ".team-dropdown__indicator path": {
+          stroke: COLORS["core-vibrant-blue-over"],
+        },
+      },
+      // TODO: Figure out a way to apply separate &:focus-visible styling
+      // Currently only relying on --is-focused styling for tabbing through app
+      ...(state.isDisabled && {
+        ".team-dropdown__single-value": {
+          color: COLORS["ui-fleet-black-50"],
+        },
+        ".team-dropdown__indicator path": {
+          stroke: COLORS["ui-fleet-black-50"],
+        },
+      }),
+      // When clicking
+      "&:active": {
+        ".team-dropdown__single-value": {
+          color: COLORS["core-vibrant-blue-down"],
+        },
+        ".team-dropdown__indicator path": {
+          stroke: COLORS["core-vibrant-blue-down"],
+        },
+      },
+      ...(state.menuIsOpen && {
+        ".team-dropdown__indicator svg": {
+          transform: "rotate(180deg)",
+          transition: "transform 0.25s ease",
+        },
+      }),
+    }),
+    singleValue: (provided) => ({
+      ...provided,
+      fontSize: "24px",
+      lineHeight: "normal",
+      paddingLeft: 0,
+      paddingRight: "8px",
+      margin: 0,
+    }),
+    dropdownIndicator: (provided) => ({
+      ...provided,
+      display: "flex",
+      padding: "2px",
+      margin: "0 5px",
+      svg: {
+        transition: "transform 0.25s ease",
+      },
+    }),
+    menu: (provided) => ({
+      ...provided,
+      boxShadow: "0 2px 6px rgba(0, 0, 0, 0.1)",
+      borderRadius: "4px",
+      zIndex: 6,
+      overflow: "hidden",
+      border: 0,
+      marginTop: 0,
+      minWidth: "330px",
+      maxHeight: "none",
+      position: "absolute",
+      left: "0",
+      animation: "fade-in 150ms ease-out",
+    }),
+    // Placeholder is never shown on teams dropdown
+    menuList: (provided) => ({
+      ...provided,
+      padding: PADDING["pad-small"],
+    }),
+    valueContainer: (provided) => ({
+      ...provided,
+      padding: 0,
+    }),
+    option: (provided, state) => ({
+      ...provided,
+      padding: "10px 8px",
+      fontSize: "14px",
+      backgroundColor: getOptionBackgroundColor(state),
+      color: COLORS["core-fleet-black"],
+      "&:hover": {
+        backgroundColor: state.isDisabled
+          ? "transparent"
+          : COLORS["ui-vibrant-blue-10"],
+      },
+      "&:active": {
+        backgroundColor: state.isDisabled
+          ? "transparent"
+          : COLORS["ui-vibrant-blue-10"],
+      },
+      ...(state.isDisabled && {
+        color: COLORS["ui-fleet-black-50"],
+        fontStyle: "italic",
+      }),
+    }),
+  };
+
   const renderDropdown = () => {
     if (teamOptions.length) {
       return (
-        <Dropdown
-          value={selectedValue}
+        <Select<INumberDropdownOption, false>
+          options={teamOptions}
           placeholder="All teams"
+          onChange={(newValue) => {
```

Comment:
Are the values here team objects rather than IDs, such that all teams/no team work correctly?

Reply:
Yeah, `newValue.value` is being typed as a number in `INumberDropdownOption`, but `newValue` itself is an object